src/Workspaces/Core/Portable/LinkedFileDiffMerging/DefaultDocumentTextDifferencingService.cs

```diff
@@ -12,7 +12,12 @@ namespace Microsoft.CodeAnalysis
     [ExportWorkspaceService(typeof(IDocumentTextDifferencingService), ServiceLayer.Default), Shared]
     internal class DefaultDocumentTextDifferencingService : IDocumentTextDifferencingService
     {
-        public async Task<ImmutableArray<TextChange>> GetTextChangesAsync(Document oldDocument, Document newDocument, TextDifferenceTypes preferredDifferenceType = TextDifferenceTypes.Word, CancellationToken cancellationToken = default)
+        public Task<ImmutableArray<TextChange>> GetTextChangesAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken)
+        {
+            return GetTextChangesAsync(oldDocument, newDocument, TextDifferenceTypes.Word, cancellationToken);
+        }
```

Comment:
FYI, this could have been done as an extension.  Which would then negate the need for both impls to provide it.  Just an FYI for the future.